src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -845,6 +845,36 @@ struct ConvertFromBinPrecision<std::tuple<src_t, dst_t>> {
     }
 };
 
+#define INTEL_CPU_CVT_FROM_2BIT_LIST                                                                 \
+    INTEL_CPU_CVT(u2, f32), INTEL_CPU_CVT(u2, f16), INTEL_CPU_CVT(u2, bf16), INTEL_CPU_CVT(u2, i32), \
+        INTEL_CPU_CVT(u2, u8), INTEL_CPU_CVT(u2, i8)
+
+struct ConvertFrom2BitContext {
+    const void* srcPtr;
+    void* dstPtr;
+    size_t size;
+    bool converted;
+};
+
+template <typename T>
+struct ConvertFrom2BitPrecision;
+
+[[maybe_unused]] static uint8_t get_u2(const uint8_t& val, uint8_t shift) {
+    return static_cast<uint8_t>((val & (0x3 << shift)) >> shift);
+}
```

Comment:
Isn't it more logical to take `val` by value rather by reference?

Reply:
Applied. Thanks Maksim!